tests/e2e/core-tests/specs/merchant/wp-admin-order-refund.test.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, */
+
+/**
+ * Internal dependencies
+ */
+const {
+	StoreOwnerFlow,
+	createSimpleProduct,
+	createSimpleOrder,
+	verifyCheckboxIsSet,
+	verifyValueOfInputField,
+	uiUnblocked,
+	addProductToOrder,
+} = require( '@woocommerce/e2e-utils' );
+
+const config = require( 'config' );
+const simpleProductName = config.get( 'products.simple.name' );
+
+let orderId;
+let currencySymbol;
+
+const runRefundOrderTest = () => {
+	describe('WooCommerce Orders > Refund an order', () => {
+		beforeAll(async () => {
+			await StoreOwnerFlow.login();
+			await createSimpleProduct();
+			orderId = await createSimpleOrder();
+			await addProductToOrder(orderId, simpleProductName);
+
+			// Get the currency symbol for the store's selected currency
+			await page.waitForSelector('.woocommerce-Price-currencySymbol')
+			let currencyElement = await page.$('.woocommerce-Price-currencySymbol')
+			currencySymbol = await page.evaluate(el => el.textContent, currencyElement)
```

Comment:
The above 3 lines are missing trailing `;`

Reply:
Ack, good catch. Fixed!